projectional/src/main/java/jetbrains/jetpad/projectional/generic/CollectionEditor.java

```diff
@@ -28,12 +29,15 @@
   private List<ViewT> myViews;
   private Property<ItemT> myForDeletion;
   private boolean myCanCreateNew;
+  private Predicate<ViewT> myReplaceWithNewOnDel;
 
-  protected CollectionEditor(List<ItemT> items, List<ViewT> views, Property<ItemT> forDeletion, boolean canCreateNew) {
+  protected CollectionEditor(List<ItemT> items, List<ViewT> views, Property<ItemT> forDeletion, boolean canCreateNew,
+      Predicate<ViewT> replaceWithNewOnDel) {
```

Comment:
Ok. I see. 
What do you think about "shouldBecomeEmpty" instead of "replaceWithNewOnDelete"?